specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databaseVulnerabilityAssessmentScans.json

```diff
@@ -0,0 +1,646 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the database vulnerability assessment scan record.",
+            "schema": {
+              "$ref": "#/definitions/VulnerabilityAssessmentScanRecord"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidStorageAccountName - The provided storage account is not valid or does not exist.\n\n * 400 InvalidStorageAccountCredentials - The provided storage account shared access signature is not valid.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 DatabaseDoesNotExist - User has specified a database name that does not exist on this server instance.\n\n * 404 SourceDatabaseNotFound - The source database does not exist.\n\n * 409 DatabaseVulnerabilityAssessmentScanIsAlreadyInProgress - Vulnerability Assessment scan is already in progress.\n\n * 500 DatabaseIsUnavailable - Loading failed. Please try again later.\n\n * 500 DatabaseVulnerabilityAssessmentOperationFailed - An error has occurred while trying to execute Vulnerability Assessment operation, please try again later"
+          }
+        },
+        "x-ms-examples": {
+          "Gets a database vulnerability assessment scan record by scan ID": {
+            "$ref": "./examples/DatabaseVulnerabilityAssessmentScanRecordsGet.json"
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "DatabaseVulnerabilityAssessmentScans"
+        ],
+        "description": "Executes a Vulnerability Assessment database scan.",
+        "operationId": "DatabaseVulnerabilityAssessmentScans_Execute",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "vulnerabilityAssessmentName",
+            "in": "path",
+            "description": "The name of the vulnerability assessment.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "x-ms-enum": {
+              "name": "VulnerabilityAssessmentName",
+              "modelAsString": true
+            }
+          },
+          {
+            "name": "scanId",
+            "in": "path",
+            "description": "The vulnerability assessment scan Id of the scan to retrieve.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully executed a Vulnerability Assessment database scan."
```

Comment:
Just confirming, is there any final response that contains the scan id? or it should always be retrieved using `DatabaseVulnerabilityAssessmentScans_ListByDatabase`?

Reply:
Currently to get the scan ID of a past scan, the user should use: DatabaseVulnerabilityAssessmentScans_ListByDatabase

If the user has a scan ID he can use DatabaseVulnerabilityAssessmentScans_Get